app/src/main/res/values/localized.xml

```diff
@@ -159,7 +159,10 @@
     <string name="p_battle_config_cmd_text_summary">Show text-box for editing Skill commands</string>
 
     <string name="p_wait_ap_regen_text">Wait for AP Regeneration</string>
-    <string name="p_wait_ap_regen_text_summary">Keeps FGA running even when AP runs out, and automatically restart node once enough AP has regenerated.</string>
+    <string name="p_wait_ap_regen_text_summary">Keeps FGA running even when AP runs out and automatically restart node once enough AP has regenerated.</string>
```

Comment:
This seems a bit awkward. How about
```suggestion
    <string name="p_wait_ap_regen_text_summary">Keep FGO active after the AP have run out. The current node will be repeated once enough AP have regenerated naturally.</string>
```

Reply:
The TW text probably also needs to be adjusted to be similar, right?

Reply:
I wrote the TW translation based on the behavior instead of a direct translation, so it goes along the lines of "after AP runs out FGA won't exit immediately, but instead will continue to check whether AP has replenished and restart node".  I think it should be ok to leave it as is...?

Reply:
I guess that's alright.